bot/cogs/groups.py

```diff
@@ -0,0 +1,77 @@
+from discord import Embed
+from discord.utils import find
+from discord.ext import commands
+
+from common.bot_logger import get_logger
+from common.helper_functions import reply_and_delete
+
+logger = get_logger(__name__)
+
+
+class GroupCog(commands.Cog, name="Group management"):
+    def __init__(self, bot):
+        self.bot = bot
+        logger.info("Group cog initialised")
+
+    @commands.group(
```

Comment:
Does this really need to be part of a command group? It is currently `!group getgroup <group> <group>` right?

Reply:
I wasn't sure where to put it but I felt like a general command group might be useful for commands we might add later (such as a command to move everyone who is in a group to a certain voice channel to start a meeting).
Currently it's `!group getgroup <group(s)>` where `<group(s)>` can be any number of groups separated by a space.

Reply:
okay that is useful indeed, but semantically it is a double command. So maybe go for `!group info <groups>`?

Reply:
Yeah I guess, maybe `!group getmembers <group(s)>`